packages/core/src/extensions/Blocks/nodes/BlockContent/ListItemBlockContent/BulletListItemBlockContent/BulletListItemBlockContent.ts

```diff
@@ -133,4 +124,6 @@ const BulletListItemBlockContent = createTipTapBlock<"bulletListItem", true>({
 export const BulletListItem = {
   node: BulletListItemBlockContent,
   propSchema: bulletListItemPropSchema,
+  toInternalHTML: serializeBlockToHTMLDefault,
```

Comment:
I don't think we want to specify this on the blocks themselves right? `toInternalHTML` should just be optional to implement I think, and if it's not implemented, use the default impl

Reply:
Imo it's actually cleaner this way since then there's no distinction between default and custom blocks in the `BlockSchema`. Because keep in mind that the default implementation between the default and custom blocks is also different (one should use `node.config.renderHTML` and the other `blockConfig.render`). I think it causes more confusion (and more logic in `createBlockSpec`) if we don't explicitly add `toInternalHTML` to the default blocks, so I think it's worth it.